flink-streaming-java/src/main/java/org/apache/flink/streaming/api/environment/StreamExecutionEnvironment.java

```diff
@@ -810,7 +811,8 @@ public RestartStrategies.RestartStrategyConfiguration getRestartStrategy() {
      * @param numberOfExecutionRetries The number of times the system will try to re-execute failed
      *     tasks.
      * @deprecated This method will be replaced by {@link #setRestartStrategy}. The {@link
-     *     RestartStrategies#fixedDelayRestart(int, Time)} contains the number of execution retries.
+     *     RestartStrategies#fixedDelayRestart(int, Duration)} contains the number of execution
+     *     retries.
```

Comment:
Not needed as well

Reply:
See my reasoning in the [above comment](https://github.com/apache/flink/pull/22984#discussion_r1441914909)